### DISCUSSION TITLE
The Witcher 3: Wild Hunt — Complete Edition

### DISCUSSION THREAD

**Discussion by iiQx2** [skyline-emu/skyline#564] [Game Requests]
;)

**Comment by Ali-sensei93** [comment]
It would be sick to have this master piece on android ready to go at all times.

**Comment by PixelyIon** [comment]
I'm very biased towards supporting this game, it would be pretty cool to have a game with the graphics it does running on a phone.

**Reply by emeliagit** [reply]
Oh yes, that would be awesome. It's also generally my favorite game, and it would be so amazing to see it lol
